charts/lakefs/templates/_env.tpl

```diff
@@ -53,6 +53,10 @@ env:
   - name: LAKEFS_AUTH_UI_CONFIG_LOGOUT_URL
     value: /logout
   {{- end }}
+  {{- if (.Values.fluffy.sso).enabled }}
```

Comment:
i think that this if is not required because there's one in [L35](https://github.com/treeverse/charts/blob/bfe7269c2399df46c7720e64a62b915849bbee48/charts/lakefs/templates/_env.tpl#L35) that wraps it and ends in [L60](https://github.com/treeverse/charts/blob/bfe7269c2399df46c7720e64a62b915849bbee48/charts/lakefs/templates/_env.tpl#L60) 

**please verify me**